site/_docs/reference.md

```diff
@@ -2362,6 +2362,7 @@ semantics.
 | o p | TO_DATE(string, format)                      | Converts *string* to a date using the format *format*
 | o p | TO_TIMESTAMP(string, format)                 | Converts *string* to a timestamp using the format *format*
 | o p | TRANSLATE(expr, fromString, toString)        | Returns *expr* with all occurrences of each character in *fromString* replaced by its corresponding character in *toString*. Characters in *expr* that are not in *fromString* are not replaced
+| m | UNCOMPRESS(binary)                             | UnCompresses binary using zlib decompression and returns the result as a string.
 | o | XMLTRANSFORM(xml, xslt)                        | Returns a string after applying xslt to supplied xml.
```

Comment:
The `m` should be in its alphabetical order.

Reply:
Alphabetical order is maintained only for Function name :)

Reply:
Personally i think each column should be in the alphabetical order, from left to right.

Reply:
Function name to be in alphabetical order makes more sense to me, I can raise separate PR if we want to change the order.